src/pages/Inventory.tsx

```diff
@@ -0,0 +1,213 @@
+        try {
+          const response = await fetch(`https://localhost:7079/api/inventory/remove/${itemId}`, {
+            method: 'DELETE',
+          });
+          if (response.ok) {
+            setItems(items.filter(item => item.id !== itemId));
+            setSelectedItem(null);
+            setItemId(null); // Reset itemId
+          } else {
+            console.error('Failed to delete item:', response.statusText);
+          }
+        } catch (error) {
+          console.error('Error deleting item:', error);
+        }
+      }
+    }
+  };
+
+  return (
+    <div id="inventory-container">
+      <div id="inventory-list-container">
+        <div id="inventory-list">
+          <h1>Inventory List</h1>
+          <ul>
+            {items.map((item, index) => (
+              <li key={index} onClick={() => handleItemClick(item)}>
+                {item.name}
+              </li>
+            ))}
+          </ul>
+          <div>
+            <input
+              type="text"
+              value={itemName}
+              onChange={handleInputNameChange}
+              placeholder="Enter Item Name"
+              className="input-item-name"
+            />
+            <button onClick={handleAddItem} className="btn-add-item">
+              Add Item
+            </button>
+          </div>
+        </div>
+      </div>
+      <div id="item-details">
+        <h1>Item Details</h1>
+        {selectedItem && <p>Selected Item: {selectedItem.name}</p>}
+        <div>
+          <div>
+            <p>About:</p>
+            <p>ID: {itemId}</p>
+            <input
+              type="text"
+              value={itemAbout}
+              onChange={handleInputAboutChange}
+              placeholder="Enter Item Info"
+              className="input-item-about"
+            />
+          </div>
+          <img src="https://via.placeholder.com/150" alt="Item Placeholder" />
+        </div>
+        <div>
+          <h3>Price: {itemPrice}</h3>
+          <h3>Quantity: {itemQuantity}</h3>
+        </div>
+        <div>
+          <label>
+            Price:
+            <input
+              type="number"
+              value={itemPrice}
+              onChange={(e) => setItemPrice(parseFloat(e.target.value))}
+              className="input-item-price"
+            />
+          </label>
+        </div>
+        <div>
+          <label>
+            Quantity:
+            <input
+              type="number"
+              value={itemQuantity}
+              onChange={(e) => setItemQuantity(parseInt(e.target.value))}
+              className="input-item-quantity"
+            />
+          </label>
+        </div>
+        <button onClick={handleSaveItem} id="btn-save-item">
+          Save Item
+        </button>
+        <button onClick={handleDeleteItem} id="btn-delete-item">
+          Delete Item
+        </button>
+      </div>
+      <div id="sales">
+        <h3>Sales?</h3>
+      </div>
+    </div>
+  );
+}
+
+interface Item {
```

Comment:
Just an idea, but what do you think about making a new folder to store this & other interfaces/classes?

Something like `Abstractions` for interfaces, `Models` for classes.

Then importing it in the relevant code when needed.